src/presenter/film-list-presenter.js

```diff
@@ -0,0 +1,26 @@
+import { render } from '../render';
+import { getContainerTemplates, getFilmSectionTemplates } from './container-template';
+import ContainerView from '../view/container-view';
+import FilmListView from '../view/film-list-view';
+import FilmCardView from '../view/film-card-view';
+import ShowMoreView from '../view/show-more-view';
+
+export default class FilmListPresenter {
+  cardLimit = 5;
+  siteContainers = getContainerTemplates();
+  filmSectionTemplates = getFilmSectionTemplates();
+  mainFilmList = new FilmListView(this.filmSectionTemplates.mainList);
+  mainFilmListContainer = new ContainerView(this.siteContainers.filmListContainer);
+
+  init = (contentSection) => {
+    this.contentSection = contentSection;
+    render(this.mainFilmList, this.contentSection);
+    render(this.mainFilmListContainer, this.mainFilmList.getElement());
+
+    for (let i = 0; i < this.cardLimit; i++) {
+      render(new FilmCardView(), this.mainFilmListContainer.getElement());
+    }
```

Comment:
где можно, избегай старых добрых for- циклов (и к слову слов else тоже избегай)
```suggestion
    const createRenderIterator = (container)=>()=> render(new FilmCardView(),container)
    Array.from({length:this.cardLimit}).forEach(createRenderIterator (this.mainFilmListContainer.getElement()))
```

Reply:
причем, обрати внимание, что функцию `createRenderIterator` можно унести в отдельный модуль и, при необходимости, тестировать ее поведение отдельно